static/src/javascripts-legacy/projects/common/modules/commercial/donot-use-adblock.js

```diff
@@ -10,7 +10,6 @@ define([
     'common/modules/adblock-banner',
     'common/modules/onward/history',
     'common/modules/ui/message',
-    'common/modules/navigation/navigation',
```

Comment:
`common/modules/navigation/navigation.js` was imported and never used by this module